study.md

```diff
@@ -27,17 +27,20 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+It's not the html I write.  It's not what you see when you "view source".
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+Yes, it's a visual representation of the DOM.
 ```
```

Comment:
Good answer. Important to understand that it is a visual representation and not the DOM itself